server/.sequelize.js

```diff
@@ -11,12 +11,17 @@ const dbConfig = {
   password: dbAuth[1],
   host: dbParams.hostname,
   port: dbParams.port,
+  ssl: false,
```

Comment:
do we want this in prod?

Reply:
by default it's false, making it to true if it's not 'localhost'. This is ok.